### PR TITLE
Add Scribe functionality

### PR DESCRIPTION
**Pull request by Ashesh3**
This pull request adds the necessary code for implementing Scribe functionality, including models, serializers, viewsets, and URL mappings.

### CODE REVIEW

**Comment by Ashesh3** [comment]
@sainak could you review this once and see if this is going in the right track as per the plugin structure?

File: care_scribe/viewsets/scribe_file.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    ScribeFileUploadCreateSerializer,
+    ScribeFileUploadUpdateSerializer,
+)
+
+
+class FileUploadFilter(filters.FilterSet):
+    file_category = filters.CharFilter(field_name="file_category")
+
+
+def check_permissions(file_type, associating_id, user):
```
Comment (vigneshhari) [inline care_scribe/viewsets/scribe_file.py:19]: Function duplicated, reuse.

File: setup.py [hunk 2]
```diff
@@ -2,43 +2,52 @@
...
-    packages=find_packages(include=['care_scribe', 'care_scribe.*']),
-    test_suite='tests',
+    keywords="care_scribe",
+    name="care_scribe",
+    packages=find_packages(include=["care_scribe", "care_scribe.*"]),
+    test_suite="tests",
     tests_require=test_requirements,
-    url='https://github.com/vigneshhari/care_scribe',
-    version='0.1.0',
+    url="https://github.com/vigneshhari/care_scribe",
```
Comment (vigneshhari) [inline setup.py:50]: Change this to ohc.

File: setup.py [hunk 3]
```diff
@@ -2,43 +2,52 @@
...
+]
 
-test_requirements = [ ]
+test_requirements = []
 
 setup(
     author="Vignesh Hari",
-    author_email='hey@vigneshhari.dev',
-    python_requires='>=3.6',
+    author_email="hey@vigneshhari.dev",
```
Comment (vigneshhari) [inline setup.py:28]: info@ohc.network

File: setup.py [hunk 4]
```diff
@@ -2,43 +2,52 @@
...
+    "django-filter",
+    "openai",
+    "jsonschema",
+]
 
-test_requirements = [ ]
+test_requirements = []
 
 setup(
     author="Vignesh Hari",
```
Comment (vigneshhari) [inline setup.py:27]: Open HealthCare Network

File: setup.py [hunk 5]
```diff
@@ -2,43 +2,52 @@
...
-with open('README.rst') as readme_file:
+with open("README.rst") as readme_file:
     readme = readme_file.read()
 
-with open('HISTORY.rst') as history_file:
+with open("HISTORY.rst") as history_file:
     history = history_file.read()
 
-requirements = [ ]
+requirements = [
```
Comment (vigneshhari) [inline setup.py:13]: Read from file ?

**Review by vigneshhari — CHANGES_REQUESTED** [review]

File: care_scribe/viewsets/scribe_file.py (hunk 1 above)
Comment (sainak) [reply care_scribe/viewsets/scribe_file.py:19]: this validation is dependent on the FileType which is different for scribe and care

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**